webpages/popup/index.js

```diff
@@ -14,6 +14,11 @@ function calculatePopupSize() {
   document.body.classList.remove("loading");
 }
 
+const commitHash = await fetch("../../.git/HEAD")
+  .then((res) => res.text())
+  .then((data) => data.replace("ref: ", ""))
+  .then((branch) => fetch(`../../.git/${branch}`).then((res) => res.text()));
```

Comment:
If a branch name contains dots or slashes, would that become a problem here? Just making sure.

Reply:
Git stores branches with slashes in subfolders so there's no problem there, I haven't tested dots yet.

Reply:
Potentially relevant documentation: https://docs.github.com/en/get-started/using-git/dealing-with-special-characters-in-branch-and-tag-names

Reply:
Seems like it's not even possible to create a branch with `..`:
![Creating a branch named foo forward slash dot dot forward slash bar](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/3fd114a2-86d5-4cec-a9bd-224a43b7744a)

Reply:
But this probably could only fetch within Scratch Addons' own directory anyway, as the protocol of the fetch is `chrome-extension://`.